eta/core/storage.py

```diff
@@ -613,6 +614,47 @@ def delete(self, file_or_folder_id):
         self._service.files().delete(
             fileId=file_or_folder_id, supportsTeamDrives=True).execute()
 
+    def delete_all_files_in_folder(self, folder_id):
```

Comment:
Why would someone want to delete all files in the folder but not the folder? Because you don't want the ID of the parent folder to change if you just deleted the entire folder with `delete()` and then recreated the empty folder?

This latter approach is likely way faster for deleting folders with lots of files in them.

Reply:
Some scenarios have been encountered where the old files were to be replaced with the new ones. Here, I wanted to remove only the files and retain the original folder to upload the new files. (doesn't want the ID of the parent folder to be changed, as this folder ID might have been mentioned elsewhere in the data source)